enterprise/app/tap/tap.tsx

```diff
@@ -610,18 +644,18 @@ export default class TapComponent extends React.Component<Props, State> {
                       }
 
                       let destinationUrl = `/invocation/${status.invocationId}?target=${encodeURIComponent(
-                        targetHistory.target.label
+                        targetHistory.target?.label || ""
                       )}`;
                       let title =
                         this.getColorMode() == "timing"
-                          ? `${this.durationToNum(status.timing.duration).toFixed(2)}s`
-                          : this.statusToString(status.status);
+                          ? `${(this.durationToNum(status.timing?.duration) || 0).toFixed(2)}s`
```

Comment:
Yeah, could simplify this by making the signature for durationToNum a little cleaner.

Reply:
Done